README.md

```diff
@@ -103,7 +103,54 @@ OpenShift LightSpeed (OLS) is an AI powered assistant that runs on OpenShift and
       ```
       **Note:** using static token will require you to set the `k8s_cluster_api` mentioned in section 6.4, as this will disable the loading of OCP config from in-cluster/kubeconfig.
 
-7. (Optional) Configure the document store
+7. Configure OLS TLS communication
+
+   This section provides instructions on configuring TLS (Transport Layer Security) for the OLS Application, enabling secure connections via HTTPS. TLS is enabled by default; however, if necessary, it can be disabled through the `dev_config` settings.
+
+
+   1. Enabling and Disabling TLS
+   
+      By default, TLS is enabled in OLS. To disable TLS, adjust the `dev_config` in your configuration file as shown below:
+
+      ```yaml
+         dev_config:
+            tls_disabled: false
+      ```
```

Comment:
`To disable TLS` it needs to be
```
    tls_disabled: true
```

Reply:
Should we call it "disable_tls" to mirror "disable_auth"?

Reply:
resolved